generators/dependencies.js

```diff
@@ -0,0 +1,14 @@
+// dependencies for generated applications
+// TO-DO: maybe add as peerDependencies for automated security checks via GitHub, then read programmatically from package.json
+export default {
+	"@sap-ux/eslint-plugin-fiori-tools": "^0.2",
+	"@sap/approuter": "^16",
```

Comment:
eventually dare to `latest`? `approuter` has been very much backwards-compatible so far...

Reply:
Agree